less/admin/AppearancePage.less

```diff
@@ -24,12 +25,17 @@
     margin-left: 10px;
   }
 
-  input {
-    float: left;
-
-    &:first-child {
-      margin-right: 2%;
-    }
+  .Form-group::after {
+    content: '';
+    display: inline-block;
+    position: absolute;
+    right: 13px;
+    bottom: 8px;
+    width: 20px;
+    height: 20px;
+    background-color: var(--input-value);
```

Comment:
I wonder if we could add some embedded SVG warning icon here as a backup for the variable based background, in case the entered value is invalid. That can act as some visual validation instead of the not so great JS validation from before.

Reply:
Hm, interesting. I mean, if we wanted validation, we can use HTML element shenanigans with dummy elements and `color` style modifications... but I do like the image part.

It could be a ::before element that is behind the element when I create the ColorInput component.

Reply:
It wouldn't even need to be that. If we just had a `background` property directly above this one, it'd be applied when the `var` holds an invalid value.

(Though I think this `background-color` may need to become `background` too, or maybe we'd have to use a slightly hacky `background-image: linear-gradient(from var(...), to var(...))` value instead.)

Reply:
So...

```css
background-image: url (/* embedded SVG */);
background-image: linear-gradient(from var(--input-value), to var(--input-value))
```

Reply:
I don't want to embed an SVG, I'd rather use the stuff already loaded like the FA icons, without duplicating their SVG data.

Reply:
Hmm yeah, in that case FA icon styles as a `::before` is the way to go, I think?

Reply:
> Hmm yeah, in that case FA icon styles as a ::before is the way to go, I think?

Or maybe have the box be a dom element and place it there with CSS, the element would contain the icon behind and flexed to be centered

Reply:
mmm flex... 🤤 

But yeah, that might be the better way to go.